src/pages/build-an-app.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Building decentralized apps
-description: Overview and guides for getting started building decentralized applications.
+title: Building apps with Stacks
+description: Resources for getting started with building apps on the network
```

Comment:
`the network` > `Stacks 2.0`?

Reply:
yes, agreed `Stacks 2.0` is better

Reply:
Is "Stacks 2.0" really a version we want to enshrine in our materials generally going forward? It seems a bit weird to put "2.0" front and center for those who aren't tracking the 1.0 vs. 2.0. difference / chain launch specifically. It'll become even more outdated as the months go on.

A newcomer developer is likely to look at "2.0" and go "huh? why 2.0?", since that version only really matters to those who're helping to prepare or track the launch of Stacks 2.0 specifically.

I could certainly see us phrasing this as "Resources for getting started with building apps on Stacks" without a version, though my hope here was to help teach that "Stacks" == "network" with the relative use of the title and description.

Reply:
Maybe we just say "on the Stacks blockchain"?

Reply:
That works for me. I'd heard before @cuevasm comment that we'd do well to use "network" going forward vs. "platform" at least. But perhaps "Stacks blockchain" is preferred to "Stacks network" these days?

Reply:
+1 try to avoid platform, it has centralized connotations. Network or blockchain is preferable!